core/src/test/java/io/onetable/TestSparkDeltaTable.java

```diff
@@ -101,7 +101,7 @@ public List<Row> insertRows(int numRows) {
   }
 
   public List<Row> insertRowsForPartition(int numRows, Integer partitionValue) {
-    return insertRowsForPartition(numRows, partitionValue, partitionField);
```

Comment:
This is changing from a field to a value, is that intentional?

Reply:
yes previously it was in error state, we need partition values in method signature.